### PR TITLE
[RSS-67] Style Dashboard (mobile)

### PR DESCRIPTION
**Pull request by comalvirdi**
The font is still Arial, I wasnt sure how we wanted to go about adding a new font to the whole site. 
Also I was having trouble making the date on the announcement card completely right aligned.

Screen Shot 2021-04-12 at 1 41 54 PM

### CODE REVIEW

File: src/components/AnnouncementCard/AnnouncementCard.jsx [hunk 1]
```diff
@@ -1,12 +1,46 @@
...
+   color: rgba(81, 40, 84, 1);
+`;
+
+const AAContainer = styled.div`
+   text-align: center;
+`;
+
+const HeaderContainer = styled.div`
+   display: flex;
+`;
```
Comment (jackfales) [inline src/components/AnnouncementCard/AnnouncementCard.jsx]:
```suggestion
const HeaderContainer = styled.div`
   display: flex;
   flex-wrap: wrap;
`;

const CardTitle = styled(Card.Title)`
   flex-basis: 50%;
`;

const CardSubtitle = styled(Card.Subtitle)`
   flex-basis: 50%;
   text-align: right;
`;
```

File: src/components/AnnouncementCard/AnnouncementCard.jsx [hunk 2]
```diff
@@ -20,21 +54,30 @@ const styleDate = (date) => {
...
+            <div>
+               <HeaderContainer>
+                  <Card.Title>
+                     <Header1>{Announcement.Announcement.title}</Header1>
+                  </Card.Title>
+                  <Card.Subtitle className="mb-2 text-muted">
+                     <Header2>
+                        {months[d.getMonth()]} {d.getDate()}, {d.getFullYear()}
+                     </Header2>
+                  </Card.Subtitle>
```
Comment (jackfales) [inline src/components/AnnouncementCard/AnnouncementCard.jsx]:
```suggestion
                  <CardTitle>
                     <Header1>{Announcement.Announcement.title}</Header1>
                  </CardTitle>
                  <CardSubtitle className="mb-2 text-muted">
                     <Header2>
                        {months[d.getMonth()]} {d.getDate()}, {d.getFullYear()}
                     </Header2>
                  </CardSubtitle>
```

File: src/components/Main/Main.jsx [hunk 3]
```diff
@@ -13,7 +13,8 @@ export default () => (
    <div className="App">
       <NavBar />
       <Switch>
-         <Route exact path="/" render={() => <LoginPage />} />
+         <Route exact path="/" render={() => <Dashboard />} />
+         {/* <Route exact path="/" render={() => <LoginPage />} /> */}
```
Comment (jackfales) [inline src/components/Main/Main.jsx]: Is there a reason you changed the routing?

File: src/components/UpcomingShiftCard/UpcomingShiftCard.jsx [hunk 4]
```diff
@@ -23,38 +41,33 @@ const styleDate = (data) => {
...
+                     {DOW[df.getDay()]}, {months[df.getMonth()]} {df.getDate()}
+                  </Header1>
+               </Card.Title>
+               <Card.Text>
+                  {df.toLocaleString('en-US', {
+                     hour: 'numeric',
+                     minute: 'numeric',
+                     hour12: true,
+                  })}
+                  -&nbsp;
```
Comment (jackfales) [inline src/components/UpcomingShiftCard/UpcomingShiftCard.jsx]: Add a space before the '-'

File: src/components/UpcomingShiftCard/UpcomingShiftCard.jsx [hunk 5]
```diff
@@ -23,38 +41,33 @@ const styleDate = (data) => {
...
+                  {dt.toLocaleString('en-US', {
+                     hour: 'numeric',
+                     minute: 'numeric',
+                     hour12: true,
+                  })}
+               </Card.Text>
+            </div>
+            <Card.Link href="#">
+               <StyledText>Details &gt;</StyledText>
```
Comment (jackfales) [inline src/components/UpcomingShiftCard/UpcomingShiftCard.jsx]: Is there a reason you removed the chevron?

File: src/components/Dashboard/Dashboard.jsx [hunk 6]
```diff
@@ -3,68 +3,120 @@ import { CardColumns, Button } from 'react-bootstrap';
 import styled from 'styled-components';
 import AnnouncementCard from '../AnnouncementCard/AnnouncementCard';
 import UpcomingShiftCard from '../UpcomingShiftCard/UpcomingShiftCard';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (jackfales) [inline src/components/Dashboard/Dashboard.jsx]: Do we need this? I thought we resolved this issue

**Review by jackfales — COMMENTED** [review]
Most of it looks good. I put it some suggestions that might fix aligning the date on the right of the AnnouncementCard. As for fonts don't worry about that for now I'm still figuring out how to put fonts across the whole application. Other than that some other clarifying comments.

File: src/components/Main/Main.jsx (hunk 3 above)
Comment (comalvirdi) [reply src/components/Main/Main.jsx]: No was just testing something and forgot to change it back, my bad

File: src/components/Dashboard/Dashboard.jsx (hunk 6 above)
Comment (comalvirdi) [reply src/components/Dashboard/Dashboard.jsx]: i've removed it, when i started working on the task the bootstrap styling was not showing up unless i added it. working fine now.

File: src/components/UpcomingShiftCard/UpcomingShiftCard.jsx (hunk 5 above)
Comment (comalvirdi) [reply src/components/UpcomingShiftCard/UpcomingShiftCard.jsx]: Some files used the chevron others used the > symbol, just wanted to maintain consistency. I can replace all with the chevron if you prefer that.

Comment (jackfales) [reply src/components/UpcomingShiftCard/UpcomingShiftCard.jsx]: Yea I would just replace with the chevron, if you could let me know what other files use the > symbol that would be great.

**Review by jackfales — APPROVED** [review]
Looks good!

**Pull request merged**